ci_framework/roles/env_op_images/tasks/main.yml

```diff
@@ -45,15 +45,31 @@
     PATH: "{{ cifmw_path }}"
   block:
     - name: Get images from the csv
-      ansible.builtin.shell: |
-        csv_name=$(oc get csv -n {{ cifmw_install_yamls_defaults['OPERATOR_NAMESPACE'] }} -l operators.coreos.com/openstack-operator.openstack-operators -o=jsonpath='{.items[*].metadata.name}')
-        oc get csv -n {{ cifmw_install_yamls_defaults['OPERATOR_NAMESPACE'] }} ${csv_name} -o=jsonpath='{.spec.install.spec.deployments[*].spec.template.spec.containers[?(@.name=="manager")].env}'
+      ansible.builtin.command: >-
+        oc get csv -n {{ cifmw_install_yamls_defaults['OPERATOR_NAMESPACE'] }}
+        -o=jsonpath='{.items[*].spec.install.spec.deployments[*].spec.template.spec.containers[?(@.name=="manager")].env}'
       register: csv_images
 
+    - name: Debug
```

Comment:
is it really needed? Patch talks about reducing output :)

Reply:
or... change task name? OK it's a copy, task name sounds wrong :)

Reply:
Maybe is not working and I'm debugging it :P